pkg/config/config.go

```diff
@@ -805,6 +805,15 @@ func findUnknownKeys(config Config) []string {
 	return unknownKeys
 }
 
+// LoadSystemProbeConfig loads in the system-probe yaml file
+func LoadSystemProbeConfig() error {
+	if err := Datadog.ReadInConfig(); err != nil {
+		log.Warnf("Error loading system probe config: %v", err)
+		return err
+	}
+	return nil
+}
```

Comment:
Can we inline this in `SetupSystemProbeConfig` ?   This function being named `LoadSystemProbeConfig` is confusing because the only reason it loads the system-probe config is when the caller points to the system-probe config file.

Reply:
Hmm, that's true. I was largely just copying what seemed to be the pattern from loading the main agent config. I don't see why there needs to have this explicitly in the config file. I'll take it out.